react/src/components/StorageStatusPanel.tsx

```diff
@@ -228,6 +233,18 @@ const StorageStatusPanel: React.FC<{
                 {t('data.ProjectFolder')}:
               </Typography.Text>
               {projectFolderCount}
+              {isAdmin &&
```

Comment:
```suggestion
              {(currentUserRole === 'admin' || currentUserRole === 'superadmin') &&
```
How about checking `isAdmin` right away for better understanding?